packages/forma-36-website/src/content/migration-v3-to-v4.mdx

```diff
@@ -50,12 +52,16 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 - **Tag** becomes Badge: We migrated the Tag component to Badge. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tag-to-badge).
 
-- **Tooltip**: Two props got renamed: `containerElement` to `as` and `place` to `placement`. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tooltip-components).
+- **Table**: Table becomes compound component in version 4. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-table-components).
 
-- **Dropdown**: The Dropdown component was removed from v4. Instead, you can use three new components: Menu, Autocomplete, or Popover. What to use depends on the specific case. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-grid-components).
+- **Tooltip**: Two props got renamed: `containerElement` to `as`; and `place` to `placement`. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tooltip-components).
 
 - **Tabs**: The Tabs components API has been improved. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tabs-components).
 
+- **Table**: The only difference in version 4 is how you import your component. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-tooltip-components).
```

Comment:
I think this is duplicated
```suggestion
```

Reply:
true, some rebasing leftovers.

Reply:
We will need a last task after to go over and be sure we didn't miss anything on the rebases 😅